src/sentry/integrations/opsgenie/client.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from sentry.shared_integrations.client.proxy import IntegrationProxyClient
+
+# from typing import Any, Mapping
+
+
+API_VERSION = "/v2"
```

Comment:
Could we maybe rename this `OPSGENIE_API_VERSION` so it's distinct from our own API?